README.md

```diff
@@ -42,6 +42,7 @@ Help welcome, eg by filling in the entries with `?` `TODO` and `CHECKME`, correc
 | User defined operators | partial: opCall opSlice, opAssign etc | yes | -1 |
```

Comment:
> many built-in types, such as integers, arrays, maps, and struct objects, can never be `null`

* maps can be null see below
* clarify that it's for fixed size arrays, but dynamic arrays can be null
```d
void main(){
  int[]x;
  assert(x is null);
  int[int]x2;
  assert(x2 is null);
}
```

Reply:
I changed it to this:

> many built-in types, such as integers, fixed-size arrays, maps, and struct objects, can never throw `null` dereference errors, either by not being `null` or by treating `null` values as empty values